src/back/dot/mod.rs

```diff
@@ -505,7 +505,11 @@ fn write_function_expressions(
                 edges.insert("reject", reject);
                 ("Select".into(), 3)
             }
-            E::Derivative { axis, expr } => {
+            E::Derivative {
+                axis,
+                ctrl: _,
```

Comment:
I think we should emit this as well.

Reply:
I'm not familiar with `dot` at all so I'd just be guessing here. If you wouldn't mind filling in this bit, everything else has been updated. 🤘